ledger/accountdb.go

```diff
@@ -5383,3 +5413,49 @@ func (qs *stateProofVerificationDbQueries) lookupData(stateProofLastAttestedRoun
 	err := db.Retry(queryFunc)
 	return &data, err
 }
+
+// stateProofVerificationTable returns all verification data currently committed to the given table.
+func stateProofVerificationTable(ctx context.Context, tx *sql.Tx, tableName string) ([]ledgercore.StateProofVerificationData, error) {
+	var result []ledgercore.StateProofVerificationData
+	queryFunc := func() error {
+		selectQuery := fmt.Sprintf("SELECT verificationdata FROM %s", tableName)
```

Comment:
I'm now sorting it after loading it from the DB, made more sense to me since I don't really need in sorted anywhere else. Take a look and let me know what you think.

Reply:
I misunderstood the reason for sorting - moved it to the db query now, as we can expect a small amount of data to sort.